openapi/features-api.yaml

```diff
@@ -0,0 +1,1166 @@
+        phq_viewership_sports_auto_racing_indy_car:
+          $ref: '#/components/schemas/PhqViewershipFeature'
+        phq_viewership_sports_baseball_ncaa:
+          $ref: '#/components/schemas/PhqViewershipFeature'
+        phq_viewership_sports_baseball_mlb:
+          $ref: '#/components/schemas/PhqViewershipFeature'
+        phq_viewership_sports_basketball_ncaa:
+          $ref: '#/components/schemas/PhqViewershipFeature'
+        phq_viewership_sports_basketball_nba:
+          $ref: '#/components/schemas/PhqViewershipFeature'
+        phq_viewership_sports_basketball_ncaa_women:
+          $ref: '#/components/schemas/PhqViewershipFeature'
+        phq_viewership_sports_golf_masters:
+          $ref: '#/components/schemas/PhqViewershipFeature'
+        phq_viewership_sports_golf_pga_championship:
+          $ref: '#/components/schemas/PhqViewershipFeature'
+        phq_viewership_sports_golf_us_open:
+          $ref: '#/components/schemas/PhqViewershipFeature'
+        phq_viewership_sports_golf_pga_tour:
+          $ref: '#/components/schemas/PhqViewershipFeature'
+        phq_viewership_sports_horse_racing_kentucky_derby:
+          $ref: '#/components/schemas/PhqViewershipFeature'
+        phq_viewership_sports_horse_racing_preakness_stakes:
+          $ref: '#/components/schemas/PhqViewershipFeature'
+        phq_viewership_sports_horse_racing_belmont_stakes:
+          $ref: '#/components/schemas/PhqViewershipFeature'
+        phq_viewership_sports_ice_hockey_nhl:
+          $ref: '#/components/schemas/PhqViewershipFeature'
+        phq_viewership_sports_mma_ufc:
+          $ref: '#/components/schemas/PhqViewershipFeature'
+        phq_viewership_sports_soccer_mls:
+          $ref: '#/components/schemas/PhqViewershipFeature'
+        phq_viewership_sports_soccer_fifa_world_cup_women:
+          $ref: '#/components/schemas/PhqViewershipFeature'
+        phq_viewership_sports_soccer_fifa_world_cup:
+          $ref: '#/components/schemas/PhqViewershipFeature'
+        phq_viewership_sports_soccer_uefa_champions_league:
+          $ref: '#/components/schemas/PhqViewershipFeature'
+        phq_viewership_sports_soccer_concacaf_champions_league:
+          $ref: '#/components/schemas/PhqViewershipFeature'
+        phq_viewership_sports_soccer_concacaf_gold_cup:
+          $ref: '#/components/schemas/PhqViewershipFeature'
+        phq_viewership_sports_soccer_copa_america:
+          $ref: '#/components/schemas/PhqViewershipFeature'
+        phq_viewership_sports_softball_ncaa_women:
+          $ref: '#/components/schemas/PhqViewershipFeature'
+        phq_viewership_sports_tennis_wimbledon:
+          $ref: '#/components/schemas/PhqViewershipFeature'
+        phq_viewership_sports_tennis_us_open:
+          $ref: '#/components/schemas/PhqViewershipFeature'
+      required:
+        - date
+      title: PhqFeatureSet
+    PhqFeaturesResponse:
+      type: object
+      properties:
+        results:
```

Comment:
Presumably we also return other pagination fields? (i know the current tech docs don't list them, they need to be added here though)